app/views/reports/twilio_numbers.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+  <div class="col-md-12">
+    <h1>Twilio numbers report request</h1>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <div class="well">
+      <%= form_for @twilio_numbers_report, url: reports_twilio_numbers_path, method: :post do |f| %>
+        <div class="form-group <%= 'form-group--error' if f.object.errors[:email].any? %>">
+          <%= f.label :email %>
+          <%= render 'error', error_messages: f.object.errors[:email] %>
+          <%= f.text_field :email, class: 'input-md-6 form-control' %>
+        </div>
+
+        * Comma separated list of users to be emailed the generated report
+        <div class="form-group">
+          <%= f.submit 'Request report', class: 'btn btn-default t-search' %>
+        </div>
+      <% end %>
+    </div>
+  </div>
+</div>
+
+<h3>Pending requests</h3>
+
+<div class="row">
+  <div class="col-md-12">
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Emails</th>
+          <th>Created At</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% GenerateTwilioNumbersReport.queued_tasks.each do |job| %>
```

Comment:
This feels kinda surprising that we're essentially interacting with a background worker in the view. I understand what the objective is but the repository-like behaviour should be exposed in something more appropriate.